frontend/app/component/form/voters_and_votes/VotersAndVotesForm.test.tsx

```diff
@@ -336,6 +334,45 @@ describe("Test VotersAndVotesForm", () => {
       expect(screen.queryByTestId("server-feedback-error")).toBeNull();
     });
 
+    test("F.13 IncorrectTotal Voters recounts", async () => {
+      overrideOnce("post", "/api/polling_stations/1/data_entries/1", 200, {
+        validation_results: {
+          errors: [
+            {
+              fields: [
+                "data.voters_recounts.total_admitted_voters_recount",
+                "data.voters_recounts.poll_card_recount",
+                "data.voters_recounts.proxy_certificate_recount",
+                "data.voters_recounts.voter_card_recount",
+              ],
+              code: "IncorrectTotal",
+            },
+          ],
+          warnings: [],
+        },
+      });
+
+      const user = userEvent.setup();
+
+      render(Component);
+
+      // TODO: These fields don't render because recounted is by default undefined
+      //  this value is coming from useState, any idea how to either refactor the code
+      //  or mock useState to be able to test this error?
```

Comment:
@lkleuver @praseodym @cikzh can any of you assist me here and in the test for W.30 below? See the TODO for the issue.

Reply:
I've solved this by making the `PollingStationFormController` more testable by allowing default values to be set through props. See the latest commit!